site/es/blog/new-in-devtools-105/index.md

```diff
@@ -29,70 +28,89 @@ _Gracias  por la traducción [Carlos Caballero](https://carloscaballero.io) y  p
 
 
 <!-- ## Step-by-step replay in the Recorder {: #recorder } -->
+## Reproducción paso a paso en la grabadora {: #recorder }
 
 <!-- You can now set a breakpoint and replay a user flow step by step in the **Recorder** panel. -->
+Ahora puede establecer un punto de ruptura y reproducir un flujo de usuario paso a paso en el panel de **Recorder**.
 
 <!-- To set a breakpoint, click on the blue dot next to a step. Replay your user flow, the replay will pause before executing the step. From here, you can continue the replay, execute a step, or cancel the replay. -->
+Para establecer un punto de ruptura, haga clic en el punto azul al lado de un paso. Reprodúzca su flujo de usuario, la reproducción se detendrá antes de ejecutar el paso. Desde aquí, puede continuar la reproducción, ejecutar un paso o cancelar la reproducción.
 
 <!-- With this feature, you can fully visualize and debug your user flow with ease. -->
+Con esta función, puede visualizar y depurar fácilmente su flujo de usuario.
 
 <!-- See [Recorder features reference](/docs/devtools/recorder/reference/) for more information. -->
+Visite [la referencia de características de la Grabadora](/docs/devtools/recorder/reference/) para más información.
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/5RqFNkPTbtEXSC4KovNF.png", alt="Step-by-step replay in the Recorder", width="800", height="547" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/5RqFNkPTbtEXSC4KovNF.png", alt="Reproducción paso a paso en la grabadora", width="800", height="547" %}
 
 Chromium issue: [1257499](https://crbug.com/1257499)
 
-
 <!-- ## Support mouse over event in the Recorder panel {: #recorder-hover } -->
+## Soporte de eventos de ratón en el panel de la grabadora {: #recorder-hover }
 
 <!-- The **Recorder** now supports adding a mouse over (hover) step manually in a recording.  -->
+La **Grabadora** ahora soporta añadir un paso de pasar el ratón por encima (hover) manualmente en una grabación.
 
 <!-- [This demo](https://jec.fyi/demo/menu-hover) shows a pop up menu on hover. Try to record a user flow and click a menu item. -->
+[Esta demo](https://jec.fyi/demo/menu-hover) muestra un menu flotante al pasar el ratón por encima. Intente grabar un flujo de usuario y haga clic en un elemento del menú.
 
 <!-- If you replay the user flow now, it will fail because the **Recorder** doesn’t capture mouse over events automatically during recording. To resolve this, [add a step manually](/docs/devtools/recorder/reference/#add-and-remove-steps) to hover over the selector before clicking the menu item.  -->
+Si reproduce el flujo de usuario ahora, fallará porque la **Grabadora** no captura automáticamente los eventos de ratón al grabar. Para resolver esto, [añada un paso manualmente](/docs/devtools/recorder/reference/#add-and-remove-steps) para pasar el ratón por encima del selector antes de hacer clic en el elemento del menú.
```

Comment:
Si reproduce el flujo de usuario ahora, fallará porque **Recorder** no captura automáticamente los eventos de ratón al grabar. Para resolver esto, [añada un paso manualmente](/docs/devtools/recorder/reference/#add-and-remove-steps) para pasar el ratón por encima del selector antes de hacer clic en el elemento del menú.